<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deriv</class>
 <widget class="QMainWindow" name="Deriv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deriv</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QDoubleSpinBox" name="h">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-108.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fx">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxm1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxM2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxm3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxM1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxm5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxm2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxm4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxM5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxM3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="fxM4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>4</number>
    </property>
    <property name="minimum">
     <double>-144.000000000000000</double>
    </property>
    <property name="maximum">
     <double>112.989999999999995</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi-2)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi+2)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi-4)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi-3)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi-5)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi+1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi-1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi+3)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi+4)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F(Xi+5)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Metodo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>Deriv</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
